docs/dev/table/connect.md

```diff
@@ -371,9 +335,23 @@ schema:
 </div>
 </div>
 
-For *each field*, the following properties can be declared in addition to the column's name and type:
+In order to declare time attributes in the schema, the following ways are supported:
 
 <div class="codetabs" markdown="1">
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+CREATE TABLE MyTable (
+  MyField1 AS PROCTIME(), -- declares this field as a processing-time attribute
+  MyField2 TIMESTAMP(3),
+  MyField3 BOOLEAN,
```

Comment:
Looks like `MyField3` not same to python and Java/Scala.
Can we do this rename by compute column?

Reply:
There is a little differentce between using computed column and `Schema#from()`. Using `Schema#from()`, users can't  reference the original field name.

Reply:
Can we do this rename by compute column?

Reply:
Yes, we can.